client/man/Makefile.am

```diff
@@ -10,7 +10,8 @@ man1_MANS = 				\
 		ipa-client-install.1	\
 		ipa-client-automount.1	\
 		ipa-certupdate.1	\
-		ipa-join.1
+		ipa-join.1		\
+		ipa.1
```

Comment:
No, setuptools no longer supports non-package data. It can no longer install files to ```PREFIX/share```, ```PREFIX/man``` or any other directory except for scripts (```PREFIX/share```) and package data.